.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Restore
         run: dotnet restore
       - name: Build
-        run: dotnet build --no-restore --framework net${{ matrix.dotnet }}
```

Comment:
@jakubfijalkowski supposedly without that flag the package is built and used locally in the CI making prerelease not needed - do you know why this flag was here in the first place?